src/messaging/ReliableMessageProtocolConfig.h

```diff
@@ -122,15 +123,23 @@ namespace chip {
  *
  */
 #ifndef CHIP_CONFIG_RMP_RETRANS_TABLE_SIZE
-#if LWIP_PBUF_FROM_CUSTOM_POOLS
-#define CHIP_CONFIG_RMP_RETRANS_TABLE_SIZE CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS
-#elif PBUF_POOL_SIZE
+#if CHIP_SYSTEM_CONFIG_USE_LWIP
+
```

Comment:
suggest `#include "lwip/opt.h"` here to ensure `PBUF_POOL_SIZE` is defined.

Reply:
Aha!  I was wondering where this sort of thing was supposed to come from.

Thank you.